web/src/views/InvestigationDetail.vue

```diff
@@ -178,12 +184,14 @@ export default defineComponent({
     const selectedDataset: Ref<Dataset | null> = ref(null);
     const rootDataset = computed(() => store.state.rootDataset);
     const tilesourceDatasets = computed(() => store.getters.tilesourceDatasets);
+    const selectionMode = computed(() => store.state.selectionMode);
     /* eslint-disable */
+    let annotationLayer: any;
```

Comment:
```suggestion
    let selectionLayer: any;
```
Since this variable is tied to the functionality for selecting a subimage, I think this name better communicates its purpose.

Reply:
Changed in [ef1705a](https://github.com/atlascope/atlascope/pull/127/commits/ef1705a3ef6f7106f6f27a43c6c39e25bbd792f7)